### PR TITLE
Reinstate etc after overwriting import.

### PR DESCRIPTION
**Pull request by plwhite**

### CODE REVIEW

File: calico/felix/test/test_felix.py [hunk 1]
```diff
@@ -39,8 +37,19 @@
...
+        global real_etcd
+        real_etcd = sys.modules.get('etcd', None)
+        sys.modules['etcd'] = stub_etcd
+
+    def tearDown(self):
+        super(TestBasic, self).tearDown()
+        if real_etcd is None:
+            sys.modules.pop('etcd')
+        else:
+            sys.modules['etcd'] = real_etcd
```
Comment (Lukasa) [inline calico/felix/test/test_felix.py]: There's no reason to use a `global` here, you can just store it on the test class itself (`self._real_etcd`).

**Comment by Lukasa** [comment]
Small note.

**Comment by plwhite** [comment]
retest this please

**Comment by Lukasa** [comment]
LGTM. :cake:

**Pull request merged**